crypto/ct/ct_log.c

```diff
@@ -243,26 +243,24 @@ int CTLOG_STORE_load_file(CTLOG_STORE *store, const char *file)
     if (load_ctx->conf == NULL)
         goto end;
 
-    ret = NCONF_load(load_ctx->conf, file, NULL);
-    if (ret <= 0) {
+    if (!NCONF_load(load_ctx->conf, file, NULL)) {
```

Comment:
This is checked as `NCONF_load(...) <= 0` elsewhere in the source. Do we know that it is boolean?